jobs/aws/eks-distro-build-tooling/prow-deck-postsubmits.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+# IMPORTANT: This Prowjob was generated by running `make prowjobs -C templater`.
+# DO NOT EDIT THIS FILE BY HAND! Refer to docs/prowjobs.md for instructions on
+# how to add a new Prowjob or update an existing Prowjob.
+################################################################################
+
+postsubmits:
+  aws/eks-distro-build-tooling:
+  - name: prow-deck-tooling-postsubmit
+    always_run: false
+    run_if_changed: "projects/kubernetes/test-infra|scripts/setup_public_ecr_push.sh"
+    branches:
+    - ^main$
+    max_concurrency: 10
+    cluster: "prow-postsubmits-cluster"
+    skip_report: false
+    decoration_config:
+      gcs_configuration:
+        bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+        path_strategy: explicit
+      s3_credentials_secret: s3-credentials
+    labels:
+      image-build: "true"
+      pr-creation: "true"
```

Comment:
this is to update the Helm chart values for Prow control plane with the new deck image.